### PR TITLE
Add tests for OracleManagerAppDataFeed

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

**Comment by jpkcambridge** [comment]
@emaG3m Looks good to me! 👍

**Comment by emaG3m** [comment]
@jpkcambridge woops, didn't see the review. Just added another commit :D

File: test/UniswapAdapter.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        uniswapExchangeMock1.address,
+        uniswapExchangeMock2.address,
+        uniswapAdapter.address
+      )
+    })
+
+    it('updates the DataFeed with the correct price', async () => {
+      const expectedResult = uintToBytes32(await getExpectedPrice(uniswapExchangeMock1, uniswapExchangeMock2))
+      await oracleManagerDataFeed.logResult();
+      expect((await oracleManagerDataFeed.resultByIndexFor(1))[0]).to.equal(expectedResult)
```
Comment (mikec) [inline test/UniswapAdapter.test.js]: could also add an expect for the "date" value .. might be hard to get the exact timestamp, so maybe just a check that it's greater than 0

**Comment by mikec** [comment]
@emaG3m just one suggestion! other than that, lookin' good 👍 👍

File: test/UniswapAdapter.test.js (hunk 1 above)
Comment (emaG3m) [reply test/UniswapAdapter.test.js]: 👍

**Pull request merged**